src/components/layouts/AdminLayout.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  Drawer,
+  List,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from "@mui/material";
+import InboxIcon from "@mui/icons-material/MoveToInbox";
+import MailIcon from "@mui/icons-material/Mail";
+import { useRouter } from "next/router";
+import { type Route } from "nextjs-routes";
+
+const DRAWER_WIDTH = 240;
+
+type MenuItem = {
+  text: string;
+  icon: JSX.Element;
+  path: "/" | Route;
+};
+
+export default function AdminLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const router = useRouter();
+
+  const menuItems: MenuItem[] = [
+    {
+      text: "Collections",
+      icon: <InboxIcon />,
+      path: "/",
+    },
+    {
+      text: "Forms",
+      icon: <MailIcon />,
+      path: "/",
+    },
+  ];
+
+  const handleNavigation = async (item: MenuItem) => {
+    const path = item.path;
+    await router.push(path);
+  };
+
+  return (
+    <div style={{ display: "flex" }}>
+      <Drawer
+        variant="permanent"
+        style={{ width: DRAWER_WIDTH }}
+        sx={{
+          width: DRAWER_WIDTH,
+          flexShrink: 0,
+          "& .MuiDrawer-paper": {
```

Comment:
Can you do this with Tailwind as well? Example https://mui.com/base-ui/guides/working-with-tailwind-css/#adding-an-interactive-slider-component